projects/Mallard/src/helpers/settings.ts

```diff
@@ -56,9 +58,29 @@ export const getSetting = (setting: keyof Settings) =>
         return unsanitize(item)
     })
 
+export const getAllSettings = async (): Promise<Settings> => {
+    const settings = await Promise.all(
+        (Object.keys(defaultSettings) as Array<
+            keyof typeof defaultSettings
+        >).map(key =>
+            getSetting(key).then(value => ({
+                key,
+                value,
+            })),
+        ),
+    )
+    return settings.reduce(
```

Comment:
can we use Object.fromEntries() for this?

Reply:
I did try to use this but our `tsconfig.json` doesn't have a new enough `lib` (I think this is ES2019) and I realised I didn't know well enough what we could and couldn't use in terms of ES versions! Probably worth me understanding that ... will come and find you 🕵